test/object_ref.jl

```diff
@@ -27,3 +29,44 @@ end
         @test obj_ref1 == obj_ref2
     end
 end
+
+@testset "Local reference counting" begin
+    local_count(o::ObjectRef) = local_count(o.oid_hex)
+    local_count(oid_hex) = first(get(Ray.get_all_reference_counts(), oid_hex, 0))
```

Comment:
Seems like this could be implemented in `ray_julia_jll` and we can have a `Ray._local_count` as well

Reply:
I moved it to test/utils.jl since it's only really used for testing